addons/xpaas-templates/xpaas-templates.addon

```diff
@@ -0,0 +1,96 @@
+# Name: xpaas
+# Description: Imports xPaaS templates
+# Url: https://github.com/jboss-openshift/application-templates/tree/ose-v1.3.6
+
+oc create -f jboss-image-streams.json -n openshift
```

Comment:
Sure, but apply seems to be the safer option in case other add-ons (custom or provided) are going to be added.